apps/www/_blog/2023-08-08-supabase-local-dev.mdx

```diff
@@ -0,0 +1,307 @@
+date: '2023-08-08'
+toc_depth: 3
+author: qiao,soedirgo,jonny
+image: launch-week-8/day-2/OG-day2.jpg
+thumb: launch-week-8/day-2/thumb-day2.jpg
+---
+
+One of our goals at Supabase is to make Postgres development delightful. To do this, we need to simplify the experience between our CLI, your code editor, and the remote Postgres database powering your applications.
+
+We received feedback recently about our local development experience, encouraging us to improve. This iteration introduces many new features to address that feedback. Let’s jump into a few of the features we’re launching today.
+
+## Postgres Language Server
+
+![Postgres Language Server](/images/blog/launch-week-8/day-2/postgres-language-server.png)
+
+One of the unique features of Supabase is the ability to access your Postgres database directly from a browser or mobile app through our [Serverless APIs](https://supabase.com/docs/guides/api). This means that developers are writing more [PL/pgSQL](https://www.postgresql.org/docs/current/plpgsql.html).
+
+While code editors have great support for most programming languages, the SQL support is underwhelming. We want to make Postgres as simple as Python. Our recently announced [Postgres Language Server](https://github.com/supabase/postgres_lsp) takes us a step in that direction - eventually it will provide first-class support for Postgres in your favorite code editor including: Linting, Syntax Highlighting, Migrations Parsing, SQL Auto-complete, and Intellisense.
+
+The Postgres Language Server is not ready for Production just yet. The majority of work is still ahead, but we've verified that the technical approach works and we're making it public now so that we can develop it in the open with input from the community. We’re already receiving amazing [feedback](https://github.com/supabase/postgres_lsp/discussions) and [support](https://news.ycombinator.com/item?id=37020610).
+
+Follow the progress of the [Postgres Language Server on GitHub](https://github.com/supabase/postgres_lsp).
+
+## Observability tools for Postgres
+
+![Inspect your database](/images/blog/launch-week-8/day-2/inspect-your-database.png)
+
+We’ve added [debugging utilities](https://supabase.com/docs/guides/database/inspect) to our CLI to identify production issues via the `supabase inspect db` command. This interface includes 19 different commands to help you solve everything from slow queries to redundant indexes.
+
+A lot of the credit for this belongs to [Heroku’s pg-extras](https://github.com/heroku/heroku-pg-extras) feature, an amazingly useful set of functionality. We’ve adapted the work they started, added a few additional commands, and made it available for any Postgres database. Simply append the `--db-url` param to use these commands with your own Postgres database.
+
+This is just a starting point for the Supabase inspector. We’ll grow this feature to become an essential part of your Postgres toolkit.
+
+<summary>
+<details>- See full command<details>
```

Comment:
@MildTomato @kiwicopple  details should go on the outside and summary here, right?

Reply:
yup - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

<img width="506" alt="image" src="https://github.com/supabase/supabase/assets/10214025/d78ab133-bfc4-4c09-9ae5-f48526f7f710">

Reply:
think notion just copied it over wrong